src/main/java/uk/gov/mint/CTHandler.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.mint;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.commons.lang3.StringUtils;
+import uk.gov.store.EntriesUpdateDAO;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+
+public class CTHandler implements Handler {
+    private final CanonicalJsonMapper canonicalJsonMapper;
+    private final String register;
+    private final String ctserver;
+    private final Client client;
+    private final EntryValidator entryValidator;
+
+    public CTHandler(String register, String ctserver, Client client, EntryValidator entryValidator) {
```

Comment:
Writing a unit test to cover this behaviour using mocks doesn't give much value. It ends up testing your implementation. I would rather write a functional test for this. The second option is to write a unit test using DropwizardClientRule, refer https://github.com/openregister/presentation/blob/master/src/test/java/uk/gov/organisation/client/GovukOrganisationClientTest.java